fetch.py

```diff
@@ -0,0 +1,166 @@
+""" A decorator to check if genome file exists or up-to-date.
+
+    Example:
+        The decorated function require the 'accession_id' kwarg
+        and optionally the 'months' kwarg::
+
+        >>> from check import genome_check
+        >>> from datetime import datetime
+
+        >>> @genome_check
+            def print_date(*args, **kwargs):
+               print "Printed by func 'print_date' ", datetime.now()
+
+        >>> print_date(accession_id='NC_000853', months=8)
+
+    Output:
+        NC_000853 File exists and created @ Tue May 17 12:48:14 2016
+        Days passed since last download 0:00:32.932481
+        Printed by func 'print_date' 2016-05-17 12:48:46.932501
+
```

Comment:
Update documentation to (a) reference the original template `check.py` for posterity and (b) include an example for this script.